6-CODE-SPLITTING/src/script1.js

```diff
@@ -1,8 +1,8 @@
-import * as _ from 'lodash';
```

Comment:
Done. All from previous points and Tree-Shakable imports used, vendors.bundle.js contains only necessary part of Lodash library